src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -127,91 +126,93 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
                     }
                 }
                 String entry = current.toString();
+                if (entry.contains("-")) {
```

Comment:
But I wouldn't extract the while loop above, because in the while loop are also correcctions, like adding space if '-' is at third position. And putting the while loop into a method that returns a boolean could have side effects, since you don't expect, that this method is changing something.